### ISSUE TITLE
Allow custom path

### ISSUE THREAD

**Issue by thenets** [mthenw/frontail#108]
I'm trying to implement the Frontail with my test environment.

Problem:
I can't expose too many ports to my team test. So I decide add paths inside the port 80 but the Frontail doesn't expected that and don't rewrite the path of JS dependencies.

I added the path "tail-archiver"
![image](https://user-images.githubusercontent.com/2138276/34743908-90e1f8e8-f572-11e7-9bb6-fe20d2b80b44.png)

And errors was raised cause don't found the dependencies:
![image](https://user-images.githubusercontent.com/2138276/34743951-bf4b6f5c-f572-11e7-91e1-323108dfebbf.png)

**Comment by mthenw** [comment]
Hey, 

what's the setup? Do you use some kind of proxy between browser and frontail?

**Comment by thenets** [comment]
@mthenw exactly. For debug of test environment I use Sentry and I'm trying to use this project to debug. I'm using the load balancer of Rancher as a proxy.

**Comment by bberenberg** [comment]
We're hoping to do something similar with nginx. In fact the hope is to run multiple of these in parallel so that each different path provides access to a different log file.

**Comment by thenets** [comment]
Yes @bberenberg. That's what I want too.

**Comment by thenets** [comment]
Yes @bberenberg. That's what I want too but using HAProxy.

**Comment by mthenw** [comment]
Closing this one as I think it should be solved as I described in https://github.com/mthenw/frontail/issues/84. Let me know what do you think!

**Comment by thenets** [comment]
@mthenw I think is different. The problem I report is related to URL.
In the current frontail release I can't set prefix in the URL:
http://localhost/ (works perfect)
http://localhost/myprefix/ (stop working)

I think is not related to #84.

**Comment by bberenberg** [comment]
Agree with the above. This one is specific to having it run with a context path, regardless of the number of instances running. #84 is about having a single instance show multiple logs.

**Comment by mzgaljic** [comment]
Can his be re-opened? Looking to have this feature as well...want to run behind an nginx proxy on a custom path.

**Comment by mthenw** [comment]
Ok, I'm reopening this issue. Will try to work on that next week.

**Comment by thenets** [comment]
Thanks @mthenw! If you want help give me the way to add this feature and I'll do what I can.

**Comment by mthenw** [comment]
@bberenberg @thenets and I have few thoughts 

If there will be additional `path` param that will cause that frontail will expose frontend on specific URL path it still doesn't solve the problem completely as you are not able to expose multiple frontail instances on the same port. 

Can you guys share me your nginx/lb configuration? I think it should be doable without adding anything in the frontail.

**Comment by bberenberg** [comment]
Just a normal reverse proxy is my goal 

```
server {
    listen www.example.com:80;
    server_name www.example.com;
    location /frontail1 {
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Server $host;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; 
        proxy_pass http://hostname:8080/;
        client_max_body_size 10M;
    }
    location /frontail2 {
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Server $host;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; 
        proxy_pass http://hostname:8081/;
        client_max_body_size 10M;
    }
}
```

**Comment by mthenw** [comment]
@bberenberg  @thenets  I just released version 4.2.0. Let me now if it works :)
